docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -2973,6 +2973,47 @@ end
 
 * https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NotToNot
 
+== RSpec/OrderedHooks
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| Yes
+| 2.2
+| -
+|===
+
+Checks that before/around/after hooks are defined in the correct order.
+
+If multiple hooks are defined in example group, they should appear in
+the following order:
+before :suite
```

Comment:
But `before(:suite)` cannot appear in an example group? https://github.com/rspec/rspec-core/blob/e95cfe3724de9b56f973e38f84fdb1b609d88e01/lib/rspec/core/hooks.rb#L197